src/utils/index.ts

```diff
@@ -18,17 +18,16 @@ import {
 import { NODE_DEFAULTS, GRID_DEFAULTS } from 'src/utils/defaults';
 
 export const clamp = (num: number, min: number, max: number) => {
+  // eslint-disable-next-line no-nested-ternary
   return num <= min ? min : num >= max ? max : num;
```

Comment:
Would be nice to fix the root issue of this one (essentially to convert this line into `ifs` so we don't have to ignore the rule in the first place), but this is a better placement of the ignore directive.